public/app/features/plugins/plugin_loader.ts

```diff
@@ -116,7 +116,7 @@ exposeToPlugin('app/core/utils/ticks', ticks);
 exposeToPlugin('app/core/config', config);
 exposeToPlugin('app/core/time_series', TimeSeries);
 exposeToPlugin('app/core/time_series2', TimeSeries);
-exposeToPlugin('app/core/table_model', TableModel);
```

Comment:
we should be very careful changing the path external things may use.  We could consider removing it, but let's not change it :)

Reply:
ahh! I see! Although at this point I am not able to run this in the UI but reverted the the change so path exposed to plugin remains same. Good catch and thank you